ui/apps/platform/cypress/integration/collections/collectionsCrudWorkflow.test.js

```diff
@@ -1,29 +1,10 @@
 import withAuth from '../../helpers/basicAuth';
-import { visitCollections } from '../../helpers/collections';
 import { hasFeatureFlag } from '../../helpers/features';
+import { tryDeleteCollection, visitCollections } from './Collections.helpers';
 
 const baseUrl = '/v1/collections';
 const autocompleteUrl = `${baseUrl}/autocomplete`;
```

Comment:
Because the only other uses of `baseUrl` were in `tryDeleteCollection` function and only use of `autocompleteUrl` is temporary intercept, what do you think about `'/v1/collections/autocomplete'` string where it is used?